### PR TITLE
Adding new blog

### PR DESCRIPTION
**Pull request by AI-Ahmed**
It is adding a new blog that summarises week 05 based on the provided materials for this week.

### CODE REVIEW

File: week05_explore/README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
   * Same topics in russian - [video](https://www.youtube.com/watch?v=WCE9hhPbCmc)
   * Note: UCB-1 is not for bernoulli rewards, but for arbitrary r in [0,1], so you can just scale any reward to [0,1] to obtain a peace of mind. It's derived directly from Hoeffding's inequality.
 
+* Very Interested blog post written by Lilian Weng that summarises this week's materials: [The Multi-Armed Bandit Problem and Its Solutions](https://lilianweng.github.io/posts/2018-01-23-multi-armed-bandit/)
```
Comment (dniku) [inline week05_explore/README.md:25]: Please replace `Interested` with `interesting`.

**Review by dniku — CHANGES_REQUESTED** [review]

File: week05_explore/README.md (hunk 1 above)
Comment (AI-Ahmed) [reply week05_explore/README.md:25]: okey!

**Review by AI-Ahmed — COMMENTED** [review]
Adjusted the Grammer mistake!

**Review by AI-Ahmed — COMMENTED** [review]
Fixed

**Pull request merged**